src/Parser/BrewmasterMonk/CombatLogParser.js

```diff
@@ -0,0 +1,66 @@
+// TODO:
+
+import React from 'react';
+
+import MainCombatLogParser from 'Parser/Core/CombatLogParser';
+
+import SuggestionsTab from 'Main/SuggestionsTab';
+import Tab from 'Main/Tab';
+import Talents from 'Main/Talents';
+
+// Core
+import HealingDone from './Modules/Core/HealingDone';
+import DamageTaken from './Modules/Core/DamageTaken';
+import DamageDone from './Modules/Core/DamageDone';
+import HealingReceived from './Modules/Core/HealingReceived';
+import Stagger from './Modules/Core/Stagger';
+import IronSkinBrew from './Modules/Spells/IronSkinBrew';
+
+// Features
+import CastEfficiency from './Modules/Features/CastEfficiency';
+import AlwaysBeCasting from './Modules/Features/AlwaysBeCasting';
+
+class CombatLogParser extends MainCombatLogParser {
+  static specModules = {
+    // Core
+    healingDone: HealingDone,
+    healingReceived: HealingReceived,
+    damageTaken: DamageTaken,
+    stagger: Stagger,
+    damageDone: DamageDone,
```

Comment:
🤔 I'm going to be moving the damage/healing/etc statistics from the core CombatLogParser to modules soon; I haven't decided on naming yet so this might break if I forget to look at this.